### PR TITLE
Ingress classname

### PR DESCRIPTION
**Pull request by PGDream**
ingress on high version，can ingressClassName attribe

### CODE REVIEW

File: charts/apisix-dashboard/templates/ingress.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ apiVersion: extensions/v1beta1
 {{- end }}
 kind: Ingress
 metadata:
+  {{- if .Values.ingress.classNmae }}
+  ingressClassName: {{- .Values.ingress.classNmae -}}
```
Comment (tokers) [inline charts/apisix-dashboard/templates/ingress.yaml:28]: There is a minimum version constraint for Kubernetes, Please update the document to classify it.

Comment (PGDream) [reply charts/apisix-dashboard/templates/ingress.yaml:28]: values.yaml  file describe "ingressClassName"  kubernetes version ?

File: charts/apisix-dashboard/values.yaml [hunk 2]
```diff
@@ -86,9 +86,16 @@ service:
 
 ingress:
   enabled: false
+  # kubernetes ingress attribe ingressClassName,default to "apisix" 
+  classNamw: "apisix"
```
Comment (tao12345666333) [inline charts/apisix-dashboard/values.yaml:90]:
```suggestion
  className: "apisix"
```

File: charts/apisix-dashboard/templates/ingress.yaml (hunk 1 above)
Comment (tao12345666333) [inline charts/apisix-dashboard/templates/ingress.yaml:28]:
```suggestion
  ingressClassName: {{- .Values.ingress.className -}}
```

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: charts/apisix-dashboard/templates/ingress.yaml (hunk 1 above)
Comment (PGDream) [reply charts/apisix-dashboard/templates/ingress.yaml]: thx, already edited;

Comment (tokers) [reply charts/apisix-dashboard/templates/ingress.yaml]: Yes, but I think readme should also be updated.

Comment (PGDream) [reply charts/apisix-dashboard/templates/ingress.yaml]: hi,readme.md already update;

**Review by tokers — APPROVED** [review]

File: charts/apisix-dashboard/values.yaml [hunk 3]
```diff
@@ -86,9 +86,17 @@ service:
 
 ingress:
   enabled: false
+  # kubernetes ingress attribe ingressClassName,default to "apisix" 
+  # Kubernetes 1.18+ support ingressClassName attribe 
+  className: "apisix"
```
Comment (tao12345666333) [inline charts/apisix-dashboard/values.yaml:91]: We should comment it by default

Comment (PGDream) [reply charts/apisix-dashboard/values.yaml:91]: why? is become kubernetes version not upgrade!?

Comment (tao12345666333) [reply charts/apisix-dashboard/values.yaml:91]: Yes, we need to maintain compatibility.

Comment (PGDream) [reply charts/apisix-dashboard/values.yaml]: ok, please reaudit

**Review by tao12345666333 — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**